spec/workflow/states/pass_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe ManageIQ::Floe::Workflow::States::Pass do
+  let(:workflow) { ManageIQ::Floe::Workflow.load(GEM_ROOT.join("examples/workflow.json")) }
```

Comment:
I know I stole this for specs - ultimately we might want an independent one, but for now this is fine.